spec/lib/miq_automation_engine/engine/miq_ae_service/miq_ae_service_rbac_spec.rb

```diff
@@ -35,7 +35,7 @@ module MiqAeServiceModelSpec
       def collect_ids_with_rbac
```

Comment:
@mkanoor Please update this method name to match the updated logic.  The upstream PR has this changed to `collect_names_with_rbac`.